org.eclipse.lsp4j.jsonrpc/src/test/java/org/eclipse/lsp4j/jsonrpc/test/ExtendableConcurrentMessageProcessorTest.java

```diff
@@ -0,0 +1,266 @@
+/******************************************************************************
+ * Copyright (c) 2016 TypeFox and others.
```

Comment:
Copyright statement should be updated for this new file (year + initial contributor affiliation).